Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/base_machineboard.yml

```diff
@@ -13,7 +13,7 @@
       tags:
         - DroneUsable
     - type: StaticPrice
-      price: 100
+      price: 85
```

Comment:
If you're making minute balance changes you need to explain why things are being changed.

Reply:
Right now the boards are an endless bag of money for Cargo. They buy steel, glass, and plastic, put in a protolat, and then print endless amounts of boards, making super profits. This is the second way to super profits after freeson. Wanted to reduce the attractiveness of this method of income a bit.